src/integration-test/java/com/commercetools/sync/integration/externalsource/producttypes/ProductTypeSyncIT.java

```diff
@@ -404,14 +404,13 @@ public void sync_WithErrorCreatingTheProductType_ShouldExecuteCallbackOnErrorAnd
         assertThat(errorMessages)
             .hasSize(1)
             .hasOnlyOneElementSatisfying(message ->
-                assertThat(message).contains("Failed to create product type of key 'key_2'.")
+                assertThat(message).contains("Failed to create draft with key: 'key_2'.")
```

Comment:
but you usually should know because you know which sync process you are running. If you are running product sync, and you get `"failed to create draft"` you know that the draft is a product. And so on, etc.. So I guess it's enough (also with the error stack trace).